vllm/model_executor/models/bart.py

```diff
@@ -599,9 +613,13 @@ def __init__(self,
             config.max_position_embeddings,
             embed_dim,
         )
-        self.layers = nn.ModuleList(
-            [BartEncoderLayer(config,cache_config,quant_config) \
-             for _ in range(config.encoder_layers)])
+        self.layers = nn.ModuleList([
+            BartEncoderLayer(config,
+                             cache_config,
+                             quant_config,
+                             prefix=f"{prefix}.layers.{_}")
+            for _ in range(config.encoder_layers)
```

Comment:
```suggestion
            BartEncoderLayer(config,
                             cache_config,
                             quant_config,
                             prefix=f"{prefix}.layers.{layer_idx}")
            for layer_idx in range(config.encoder_layers)
```

Reply:
Similar for the other encoder blocks.

Reply:
fixed in [19c70df](https://github.com/vllm-project/vllm/pull/10558/commits/19c70df43e3f58ef420e6ec0436a0c45cc146070)